src/LinkDotNet.Blog.Domain/BlogPost.cs

```diff
@@ -35,6 +38,35 @@ private BlogPost()
 
     public string TagsAsString => Tags is null ? string.Empty : string.Join(", ", Tags);
 
+    public string Slug => GenerateSlug();
+
+    private string GenerateSlug()
+    {
+        // Remove all accents and make the string lower case.
+        if (string.IsNullOrWhiteSpace(Title))
+            return Title;
+
+        Title = Title.Normalize(NormalizationForm.FormD);
+        char[] chars = Title
+            .Where(c => CharUnicodeInfo.GetUnicodeCategory(c)
+            != UnicodeCategory.NonSpacingMark).ToArray();
```

Comment:
Super small nitpick: I like to have the `ToArray` on a separate line.

PS: I like the approach with NormaliziationForm - very sneaky way! I like

Reply:
Also I prefer `var` - not that I have a strong opinion on what to use, but I like to keep it consistent throughout the code.